composer.json

```diff
@@ -14,22 +14,31 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php-http/discovery": "^0.2.0",
+        "php": ">=5.5",
+        "php-http/discovery": "^0.6.1",
```

Comment:
this would now need to be 0.6.3.

but we have the same discussion as in https://github.com/FriendsOfSymfony/FOSHttpCache/issues/268 - maybe we should not depend on discovery and explain to add a client and either be explicit or additionally add discovery. and adjust our code to nicely report when discovery is missing and nothing was configured.

okay, but that is a bit out of scope for this PR, if you can upgrade to 0.6.3 for now, that should hopefully fix the build.